impeller/aiks/aiks_unittests.cc

```diff
@@ -1636,7 +1636,7 @@ TEST_P(AiksTest, ColorWheel) {
   auto callback = [&](AiksContext& renderer) -> std::optional<Picture> {
     // UI state.
     static bool cache_the_wheel = true;
-    static int current_blend_index = 3;
+    static int current_blend_index = 14;
```

Comment:
Nit: Keep default to SourceOver? Although advanced blends are more contentious, so maybe someone flipping through is more likely to notice a problem with Screen as the default.

Reply:
Yeah, just thought defaulting to the advanced ones was more likely to throw up issues.